src/components/dashboard/sidebar/CouponCodesSummaryCard.jsx

```diff
@@ -24,24 +19,32 @@ const CouponCodesSummaryCard = ({
     couponCodeWarningModalOpen,
     onCouponCodeWarningModalClose,
   ] = useToggle(false);
-
+  const intl = useIntl();
   const badgeVariantAndLabel = useMemo(() => {
     if (couponCodesCount > 0) {
       return ({
         variant: COUPON_CODES_AVAILABLE_BADGE_VARIANT,
-        label: COUPON_CODES_AVAILABLE_BADGE_LABEL,
+        label: intl.formatMessage({
+          id: 'enterprise.dashboard.sidebar.coupon-codes.available-badge.label',
```

Comment:
Can we please rename the id to  'enterprise.dashboard.sidebar.coupon.codes.available.badge.label' to be consisted with our id naming convention?